gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -107,25 +99,26 @@ def extract_freq_info(
         combine_operator="or",
         exact_match=True,
     )
+    logger.info("Filtering to only adj frequencies for FAF...")
     faf_meta, faf = filter_arrays_by_meta(
         hl.literal(faf_meta),
         {"faf": faf["faf"]},
-        {"gen_anc": faf_pops},
+        {"group": ["adj"]},
         combine_operator="or",
     )
 
-    # Rename filtered annotations with supplied prefix and remove "raw" group from faf.
+    # Rename filtered annotations with supplied prefix.
     ht = ht.select(
         **{
             f"{prefix}_freq": array_exprs["freq"],
-            f"{prefix}_faf": ht.faf[:1].extend(faf["faf"][2:]),
+            f"{prefix}_faf": faf["faf"][2:],
```

Comment:
```suggestion
            f"{prefix}_faf": faf["faf"],
```
I think this is right or maybe I'm misunderstanding?